examples/complete/emr-cluster.tf

```diff
@@ -31,10 +31,11 @@ module "emr" {
     "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy"
   ]
   bootstrap_actions = [
+    #Workaround for running scripts from an S3 bucket from outside of us-east-1 on the complete example.
```

Comment:
Could you describe in more detail why the workaround was necessary?

Reply:
Sure! We added a comment on [f36764e](https://github.com/Datatamer/terraform-aws-tamr-config/pull/35/commits/f36764e91ccec60305664a8d3c698f6ea8c0c39e)